generic-sensor/generic-sensor-tests.js

```diff
@@ -120,7 +169,7 @@ function runGenericSensorTests(sensorType) {
     sensor.stop();
   }, `${sensorType.name}: sensor.start() returns undefined`);
 
-  promise_test(async t => {
+  sensor_test(async t => {
```

Comment:
Actually many tests can be automated without Test API. We should only replace those tests really need Test API for automation.